feat(ecs): enable cluster to grant task protection API permissions to IAM entities (#28486)
aws/aws-cdk: packages/aws-cdk-lib/aws-ecs/README.md

)
```

> The grantTaskProtection method simplifies the process of granting ECS Task Protection API permissions to a specified IAM entity. This is particularly useful for scenarios where you need to allow an IAM role or user to manage task protection settings in an ECS cluster. By using this method, you can efficiently assign the ecs:UpdateTaskProtection permission to the desired entity, ensuring it has the necessary rights to interact with task protection features.

Reviewer: Could you please insert appropriate line breaks?

Reviewer: Also, it might be a good idea to enclose CDK-specific words in backquotes, like `grantTaskProtection` and `ecs: UpdateTaskProtection`.

Reviewer: The explanation could be simplified a bit more.
For example, as follows.

```suggestion
To manage task protection settings in an ECS cluster, you can use the `grantTaskProtection` method.
This method grants the `ecs:UpdateTaskProtection` permission to a specified IAM entity.
```
